### DISCUSSION TITLE
Minus sign instead of hyphen for negative numbers

### DISCUSSION THREAD

**Discussion by vanillajonathan** [minhhungit/ConsoleTableExt#11] [Ideas]
Example `−123` instead of `-123`. This would be more typographically correct, and would make it easier to discern negative numbers from positive number.

Unicode symbol: U+2212

**Comment by minhhungit** [comment]
I have no idea about this problem, honestly I am not sure if it is problem actually, my lack of knowledge about typographically.

IMH I think it is normal because we don't change font of the minus char, it has same font with number/text (default console font)

Anyway, this kind of problem can be resolved using [Formatter ](https://github.com/minhhungit/ConsoleTableExt/discussions/14#discussioncomment-352757)

**Reply by vanillajonathan** [reply]
Yeah, I think this can be solved using formatter.

```cs
builder.WithFormatter(col_index, (text) => text.StartsWith('-') ? text.Replace('-', '−') : text);
```
